mock/environment/__init__.py

```diff
@@ -16,34 +17,49 @@
 class Env:
    
     def __init__(self):
+        """
+        Stores weather data into a dictionary 
+        that can be indexed by site and date.
+        """
         self.site_data = {}
```

Comment:
This confuses me, because we don't need the `site_data` stored in here.
We just need the `site_data_by_night`.